src/components/Deployment/Deployment.tsx

```diff
@@ -197,23 +197,36 @@ const Wrapper = styled.div`
 
 const Section = styled.div`
   display: flex;
-  background: ${({ theme }) => theme.color.background.topMenu};
-  margin-top: 5px;
-  padding: 15px;
+  background: ${({ theme }) => theme.color.background.card};
+  margin-top: 8px;
+  padding: 10px;
   border-radius: 10px;
-  font-weight: bold;
+  font-family: 'PTRootUIWebMedium', sans-serif;
+  box-shadow: 0 2px 8px 0 rgba(26, 23, 38, 0.3);
 `;
 
 const Header = styled.div`
   display: block;
   color: ${({ theme }) => theme.color.text.outerLabel};
+  font-size: 15px;
+`;
+
+const Label = styled.label`
+  display: inline-block;
+  padding: 0;
+`;
+
+const HorizontalLine = styled.div`
+  width: 100%;
+  height: 0.1px;
+  background: ${({ theme }) => theme.color.text.outerLabel};
 `;
 
 const Body = styled.div`
   display: block;
   overflow-y: scroll;
   background-color: ${({ theme }) => theme.color.background.listItem};
-  max-height: 200px;
+  max-height: 293px;
```

Comment:
Is this height from Zeplin? Feels very specific

Reply:
> Based on the screenshots, the text alignment seems off - could you try centering the text + icons vertically as well in this PR?

yes sure!